### ISSUE TITLE
Are the quantized models differentiable

### ISSUE THREAD

**Issue by SwapnilDreams100** [hahnyuan/PTQ4ViT#6]
Hi,
Are the quantized models differentiable(i.e. can we get gradients using backprop using these) or is it not possible due to actual INT8 quantization? Pl reply ASAP

**Comment by SuperVan-Young** [comment]
The quantized models are not differentiable. We use `round_()` in pytorch to quantize, and it blocks gradients from backpropagation.

**Comment by SwapnilDreams100** [comment]
Got it thanks
